src/common/hooks/useTypes/useTypes.js

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+import { TYPES_API } from '../../constants';
+
+export const useTypes = () => {
+  const ky = useOkapiKy();
+
+  const searchParams = {
+    limit: LIMIT_MAX,
+    query: 'cql.allRecords=1 sortby name',
+  };
+
+  const { isLoading, data } = useQuery(
+    ['ui-organizations', 'organization-types'],
+    () => {
+      return ky.get(TYPES_API, { searchParams }).json();
+    },
+  );
+
+  return ({
+    organizationTypes: data?.organizationTypes,
```

Comment:
```suggestion
    organizationTypes: data?.organizationTypes ?? [],
```

Reply:
with changing `organizationTypes: data` to `organizationTypes: data?.organizationTypes ?? [],` the `result.current.organizationTypes` in the useTypes.js is either undefined or an empty array. 
@AndreiShumski2  Do you know how to fix that?

Reply:
@AndreiShumski2  just found a solution